src/WebJobs.Extensions.MobileApps/WebJobs.Extensions.MobileApps.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Microsoft.Azure.WebJobs.Extensions.MobileApps</AssemblyName>
+    <RootNamespace>Microsoft.Azure.WebJobs.Extensions.MobileApps</RootNamespace>
+    <PackageId>Microsoft.Azure.WebJobs.Extensions.MobileApps</PackageId>
+    <Authors>Microsoft</Authors>
+    <Company>Microsoft</Company>
+    <PackageRequireLicenseAcceptance>False</PackageRequireLicenseAcceptance>
+    <Product>Microsoft.Azure.WebJobs</Product>
+    <Version>3.0.0-beta1$(VersionSuffix)</Version>
+    <Description>Microsoft Azure WebJobs SDK MobileApps Extensions</Description>
+    <Copyright>© .NET Foundation. All rights reserved.</Copyright>
+  </PropertyGroup>
```

Comment:
What are the right style cop directives to add? Doesn't look like any static analysis is configured for the projects in this repo.

Reply:
Add this import to the project:
https://github.com/Azure/azure-webjobs-sdk-extensions/blob/dev/src/WebJobs.Extensions.Http/WebJobs.Extensions.Http.csproj#L2

Common props will set copyright notices, stylecop rules, version, all the common stuff. The only thing needed in the actual project file (for project info) is the assembly name and default namespace.

Reply:
I already have that import

Reply:
Odd... didn't notice it there when I looked. The duplicate properties should be removed from here so we can manage them all in the same spot.